docs/source/serialization.mdx

```diff
@@ -109,7 +109,7 @@ which should show the following logs:
 
 ```bash
 Validating ONNX model...
-        -[✓] ONNX model outputs' name match reference model ({'last_hidden_state'})
+        -[✓] ONNX model output names match reference model ({'last_hidden_state'})
```

Comment:
This fixes a problem with syntax highlighting in the frontend


![Screen Shot 2022-01-12 at 11 19 32](https://user-images.githubusercontent.com/26859204/149121880-0af7e6e1-6fa2-4fca-89f9-8348d35fa70b.png)